selenium/che-selenium-core/bin/webdriver.sh

```diff
@@ -680,14 +692,15 @@ runTests() {
 
     mvn clean verify -Pselenium-test \
                 ${TESTS_SCOPE} \
-                -Dhost=${PRODUCT_HOST} \
+                -Dche.host=${PRODUCT_HOST} \
                 -Dche.port=${PRODUCT_PORT} \
-                -Dprotocol=${PRODUCT_PROTOCOL} \
+                -Dche.protocol=${PRODUCT_PROTOCOL} \
                 -Ddocker.interface.ip=$(detectDockerInterfaceIp) \
                 -Ddriver.port=${WEBDRIVER_PORT} \
                 -Ddriver.version=${WEBDRIVER_VERSION} \
                 -Dbrowser=${BROWSER} \
-                -Dthreads=${THREADS} \
+                -Dche.threads=${THREADS} \
+                -Dche.workspace.pool.size=${WORKSPACE_POOL_SIZE} \
```

Comment:
The previous name I liked more )
We have a convention to use dots to show components separation and underscores to separate words in one component. 
So, soncider: `che.workspace_pool_size`

Reply:
But [SeleniumTestConfiguration](https://github.com/eclipse/che/blob/5.19.0/selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/configuration/SeleniumTestConfiguration.java#L120) replaces "_" on "." in module, so "che.workspace.pool.size" is just an exact name which we should inject into classes.